forms-flow-data-analysis-api/Dockerfile

```diff
@@ -11,17 +10,24 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 COPY requirements.txt .
 ENV PATH=/venv/bin:$PATH
-ARG MODEL_ID=Seethal/sentiment_analysis_generic_dataset
+# ARG MODEL_ID=Seethal/sentiment_analysis_generic_dataset
+ARG MODEL_ID=distilbert-base-uncased-finetuned-sst-2-english
```

Comment:
so here we changed the model right and based on that the size reduced right?

Reply:
Both model and changed pytorch to use cpu based dependency.